test/extension/dynamic_image/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@
 #
 message(STATUS "Boost.GIL: Configuring tests in test/extension/dynamic_image")
 foreach(_name
+  any_image
```

Comment:
Could you also add `run any_image.cpp ;` above this line in the `Jamfile`?

https://github.com/boostorg/gil/blob/9d0d22638eacbe6a7a6a8404c4586e572c6807fe/test/extension/dynamic_image/Jamfile#L13

Reply:
Done